### PR TITLE
Draft: Multi arch build

### PR DESCRIPTION
**Pull request by stmh**
* Move base image to debian:buster
* fix wkhtmltopdf installer, use build arg for architecture
* first draft of multi-arch-build-script, can be used also to build all images for native platform

For multi-arch-build you need a registered multi-arch-builder:

```
docker buildx create --name mybuilder
docker buildx use mybuilder
```

TODO: 
* Build script and `buildx` does not use already built docker images, instead pulls outdated from the registry, 
* script does not make sense yet for arm64, and amd64.

### CODE REVIEW

**Comment by mikran** [comment]
What is the current status of this, what needs to be done?

**Comment by stmh** [comment]
hi @mikran 

This needs to be tested. Use the tag `php74-multi-arch` or `php80-multi-arch` to see if everything is working correctly. Thn teh wkhtmltopdf needs to be fixed an we can merge back.

**Comment by stmh** [comment]
(the tag names are temporary for now, after testing and merging the standard tags will be used)

File: .github/workflows/php74-github-actions.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Build php74 docker images multi-arch
+
+on:
+  push:
```
Comment (mikran) [inline .github/workflows/php74-github-actions.yml:4]: instead of push this should be something like 

```
    schedule:
      interval: "daily"
```

Comment (stmh) [reply .github/workflows/php74-github-actions.yml:4]: good idea!

**Comment by stmh** [comment]
I think there is still a flaw in the xdebug images, as they are based on `php-74` and not on `php74-multi-arch` i'd suggest to use a build argument to pass the exact tag to the dependant dockerfile. This should also fix the wkhtmltopdf docker images. Can you readd them?

@i1337m an you work in this?

**Comment by i1337m** [comment]
@stmh  sure, I'll take a look on it.

**Comment by i1337m** [comment]
@mikran is it possible please to check if the xdebug, are working correctly? 
thanks

**Comment by stmh** [comment]
closing in favor of #14

**Pull request closed without merging**